kie-soup-maven-utils/kie-soup-maven-integration/src/test/java/org/appformer/maven/integration/embedder/MavenProjectLoaderTest.java

```diff
@@ -16,83 +16,143 @@
+        String depPomURL = MavenProjectLoaderTest.class.getResource("/assets/depArtifactId.pom").getFile();
+
+        String depPomURLWithTransitive = MavenProjectLoaderTest.class.getResource("/assets/depArtifactId-transitive.pom").getFile();
+
+        RepositorySystemSession session = Aether.getAether().getSession();
+        RepositorySystem repo = Aether.getAether().getSystem();
+
+        InstallRequest installRequest = new InstallRequest();
+
+        installRequest.addArtifact(new DefaultArtifact(GROUP_ID,
+                                                       TRANSITIVE_ARTIFACT_ID,
+                                                       "",
+                                                       "jar",
+                                                       VERSION,
+                                                       Collections.emptyMap(),
+                                                       new File(depJarURL)));
+
+        installRequest.addArtifact(new DefaultArtifact(GROUP_ID,
+                                                       DEP_ARTIFACT_ID,
+                                                       "",
+                                                       "pom",
+                                                       VERSION,
+                                                       Collections.emptyMap(),
+                                                       new File(addTransitive ? depPomURLWithTransitive : depPomURL)));
+
+        installRequest.addArtifact(new DefaultArtifact(GROUP_ID,
+                                                       DEP_ARTIFACT_ID,
+                                                       "",
+                                                       "jar",
+                                                       VERSION,
+                                                       Collections.emptyMap(),
+                                                       new File(depJarURL)));
+
+        repo.install(session, installRequest);
     }
 
     static String settings(String localRepo) {
-        return  "<settings xmlns=\"http://maven.apache.org/SETTINGS/1.0.0\"\n" +
-                "      xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\"\n" +
-                "      xsi:schemaLocation=\"http://maven.apache.org/SETTINGS/1.0.0\n" +
-                "                          http://maven.apache.org/xsd/settings-1.0.0.xsd\">\n" +
-                "     <localRepository>" + localRepo + "</localRepository>" + 
-                "</settings>";
+        return "<settings xmlns=\"http://maven.apache.org/SETTINGS/1.0.0\"\n" +
+               "      xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\"\n" +
+               "      xsi:schemaLocation=\"http://maven.apache.org/SETTINGS/1.0.0\n" +
+               "                          http://maven.apache.org/xsd/settings-1.0.0.xsd\">\n" +
+               "     <localRepository>" + localRepo + "</localRepository>" +
+               "</settings>";
```

Comment:
format changes?

Reply:
Hello @porcelli 

That's correct, that was my formatter messing with the XML format - but no change to the content itself